viewer/src/viewer.js

```diff
@@ -250,8 +290,58 @@ function main() {
         });
     }
 
-    var viewer = new Viewer();
-
+    anim = document.getElementById("anim");
+    anim.onmousedown = function(e) {
+        // make sure that mouse actions on the <div id="anim"> don't manipulate the orbit camera
+        e.preventOrbit = true;
+    }
+    
+    anim_select = document.getElementById("anim_select");
+    anim_select.onchange = function(e) {
+        const clipName = anim_select.value;
+        const clip = viewer.gltf.animComponent.animClipsMap[clipName];
+        anim_info.innerHTML = clip.duration + "s " + clipName;
+        anim_slider.max = clip.duration;
+        viewer.gltf.animComponent.curClip = clipName;
+        viewer.pauseAnimationClips();
+        viewer.playCurrentAnimationClip();
+    }
+    anim_select.onclick = function(e) {
+        console.log(e, anim_select.value);
+    }
+    
+    anim_slider = document.getElementById("anim_slider");
+    anim_slider.oninput = function(e) {
+        const curTime = anim_slider.value;
+        // once we seek into the animation, stop the default playing
+        viewer.pauseAnimationClips();
+        // now set the seeked time for the last played clip
+        const clip = viewer.gltf.animComponent.getCurrentClip()
+        const session = clip.session;
+        const self = session;
+        session.curTime = curTime;
+        self.showAt(self.curTime, self.fadeDir, self.fadeBegTime, self.fadeEndTime, self.fadeTime);
+        self.invokeByTime(self.curTime);
+    }
+    
+    anim_pause = document.getElementById("anim_pause");
```

Comment:
Where is this declared?

Reply:
Just there, so I could use it easily from devtools, gonna make it an attribute of Viewer